src/client/Microsoft.Identity.Client/AuthenticationResult.cs

```diff
@@ -37,7 +38,7 @@ public partial class AuthenticationResult
         /// <param name="correlationId">The correlation id of the authentication request</param>
         /// <param name="tokenType">The token type, defaults to Bearer. Note: this property is experimental and may change in future versions of the library.</param>
         /// <param name="authenticationResultMetadata">Contains metadata related to the Authentication Result.</param>
-        public AuthenticationResult(
+        public AuthenticationResult( // for backwards compat with 4.17+
```

Comment:
>/ for backwards compat with 4.17+ [](start = 38, length = 33)

This should probably say compat with 4.17 to 4.20.1, and recommended ctor ?

Reply:
Correct.

Reply:
Still missing messaging around compat with 4.20.2+?

---
In reply to: [504958399](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2096#discussion_r504958399) [](ancestors = 504958399)

Reply:
Also, I think this should not be a code comment, but a remark. Recommended ctor or something similar?

---
In reply to: [505033029](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2096#discussion_r505033029) [](ancestors = 505033029,504958399)